init.sql

```diff
@@ -0,0 +1,26 @@
+CREATE ROLE librarian WITH LOGIN SUPERUSER PASSWORD 'librarian';
+ALTER DATABASE library OWNER TO librarian;
+
+CREATE TABLE person (
+    id          INT         PRIMARY KEY,
+    firstName   VARCHAR(32),
+    lastName    VARCHAR(32),
+    birthdate   DATE,
+    hometown    VARCHAR(32)
+);
+
+CREATE TYPE phone_type AS ENUM ('work', 'home', 'personal');
+CREATE TABLE phone (
+    number      INT         PRIMARY KEY,
+    type        phone_type,
+    person_id   INT,
+    FOREIGN KEY (person_id) REFERENCES  person(id)
+);
+
+CREATE TYPE address_type AS ENUM ('work', 'home', 'summerhouse');
+CREATE TABLE address (
+    address     VARCHAR(64) PRIMARY KEY,
```

Comment:
The same here don't use address as primary key, because, for example, work address can be one for many. And else one point is bad idea to use varchar as `Primary Key`, because, it create very big and slow index, especially you'll see it if you try to use this type of the `Primary Key` with relation it'll spend many space to save `Foreign Key`.